src/CFBG.cpp

```diff
@@ -72,6 +77,35 @@ uint32 CFBG::GetBGTeamAverageItemLevel(Battleground* bg, TeamId team)
     return Sum / Count;
 }
 
+uint32 CFBG::GetBGTeamSumPlayerLevel(Battleground* bg, TeamId team)
+{
+    if (!bg)
+    {
+        return 0;
+    }
+
+    uint32 PlayersCount = bg->GetPlayersCountByTeam(team);
+    if (!PlayersCount)
+    {
+        return 0;
+    }
```

Comment:
why do we need this block?

Reply:
avoid going forward with the execution if there are no players

Reply:
@Helias I don't get it: below you are initialising `sum` to 0, then there is a for loop, then you return `sum`. If there are no players, the for loop won't begin and you'll return 0 anyway. So back to my original question: why do we need this block (lines 87-91)?

Reply:
removed